src/frontend/pages/login/login.jsx

```diff
@@ -1,8 +1,50 @@
 import React from "react";
+import { Navbar } from "../../components/navbar/Navbar";
+import { Link } from "react-router-dom";
 
 export function Login(){
+    
 
     return (
-        <h1> login page </h1>
+        <>
+            <Navbar />
+            <div className="login-card flex-col">
+                <h1 className="text-center margin-zero">Login</h1>
+                <form className="flex-col">
+                    <div className="flex-col">
+                        <label className="larger-font" for="email">Email address</label>
+                        <input className="input padding-top-12px" type="email" placeholder="abc@gmail.com" required/>
+                    </div>
+
+                    <div className="flex-col" >
+                        <label className="larger-font" for="password">Password</label>
+                        <input className="input  padding-top-12px" placeholder="enter your password" id='password' type="password" required/>
+                    </div>
+                    <div className="flex-col-login margin-top-8px">
+                        <div className="flex-row-login">
+                            <label className="flex-row-login">
+                                <input className="input-checkbox" type="checkbox" name="light"  value=""/>
+                                <span>Remember me</span>
+                            </label>
+                            <Link  className="link-forgot-pass" to="#">Forgot password</Link>
```

Comment:
this could do with a font size one stop smaller , and usually forgot password goes with a "?" on the end , just a suggestion.

Reply:
> UI looks super cool and clean!
> 
> Hoping to see future versions of this project soon.
> 
> suggestions : your navbar colors and auth buttons colors dont match hope that is intentional. fix your deploy link , doesnt seem to work. there is a screenshotting tool as chrome extension called go fullpage , you can try using that to screenshot the entire scree.

thanks for review and suggestion and yes there were few fix in CSS i kept those  pending bcos going to fix those later.